test/service/test_app_logging.py

```diff
@@ -106,6 +106,15 @@ def filter_body(organ: str) -> JSON:
                     **AzulChaliceApp.security_headers(),
                     'Cache-Control': 'no-store'
                 }
+                expected_body_log = f'… with a request body of length {len(request_body)} '
+                if debug == 0:
+                    expected_body_log += f'and type {type(request_body)}'
+                elif debug == 1 and len(request_body) > prefix_len:
+                    prefix_len_without_tail = prefix_len - len(tail := b'...')
+                    request_body_with_tail = request_body[:prefix_len_without_tail] + tail
+                    expected_body_log += f'starting in {request_body_with_tail}'
```

Comment:
This might contradict my previous advice, but looking at the new code, I think I prefer

```suggestion
                    tail = b'...'
                    request_body_prefix = request_body[:prefix_len - len(tail)]
                    expected_body_log += f'starting in {request_body_prefix}{tail}'
```

Reply:
So `{request_body_prefix}{tail}` [doesn't quite work](https://github.com/DataBiosphere/azul/actions/runs/18318735676/job/52166040095#step:5:13218), the ellipsis is withing the quoted b'*…' string in the logs.